code/API_definitions/qod-provision.yaml

```
@@ -0,0 +1,876 @@
+openapi: 3.0.3
+info:
+  title: QoD Provisioning API
+  description: |
+    The Quality-On-Demand (QoD) Provisioning API offers a programmable interface for developers to request the assignment of a certain QoS Profile to a certain device, indefinitely.
+
+    This API sets up the configuration in the network so the requested QoS profile is applied to an specified device, at any time while the provisioning is available. The device traffic will be treated with a certain QoS profile by the network whenever the device is connected to the network, until the provisioning is revoked.

```

```suggestion
    This API sets up the configuration in the network so the requested QoS profile is applied to an specified device, at any time while the provisioning is available. The device traffic will be treated with a certain QoS profile by the network whenever the device is connected to the network, until the provisioning is deleted.

```

The description says revoked, but the API call is delete to `Release resources related to QoS provisioning.` The wording in the description should be consistent with the API.  If we need to support a case where an operator revokes the QoS provisioning on a device, then we should to be clear about that.